pydantic/typing.py

```diff
@@ -399,7 +399,10 @@ def resolve_annotations(raw_annotations: Dict[str, Type[Any]], module_name: Opti
             else:
                 value = ForwardRef(value, is_argument=False)
         try:
-            value = _eval_type(value, base_globals, None)
+            if sys.version_info > (3, 13):
+                value = _eval_type(value, base_globals, None, type_params=())
```

Comment:
I assume we're fine with the new type param syntax just not working in v1?

Reply:
Yep, correct!

Reply:
@vfazio, to your point from earlier, this is more of a bandaid and not an actual fix (we're not really adding support for 3.13, just ensuring it doesn't break things)